yarn.lock

```diff
@@ -3590,6 +3590,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask-institutional/portfolio-dashboard@npm:1.1.2":
```

Comment:
@zone-live did you also run lavamoat and patches?

Reply:
I did run `yarn allow-scripts auto` and `yarn lavamoat:auto`. Is it enough?